src/main.rs

```diff
@@ -49,3 +51,29 @@ fn run() -> Result<(), failure::Error> {
     run_wasm_pack(args.cmd)?;
     Ok(())
 }
+
+fn setup_panic_hooks() {
+    let meta = human_panic::Metadata {
+        version: env!("CARGO_PKG_VERSION").into(),
+        name: env!("CARGO_PKG_NAME").into(),
+        authors: env!("CARGO_PKG_AUTHORS").replace(":", ", ").into(),
+        homepage: env!("CARGO_PKG_HOMEPAGE").into(),
+    };
+
+    let default_hook = panic::take_hook();
+
+    match env::var("RUST_BACKTRACE") {
+        Err(_) => {
+            panic::set_hook(Box::new(move |info: &panic::PanicInfo| {
+                // First call the default hook that prints to standard error.
+                default_hook(info);
+
+                // Then call human_panic.
+                let file_path = human_panic::handle_dump(&meta, info);
+                human_panic::print_msg(file_path, &meta)
+                    .expect("human-panic: printing error message to console failed");
```

Comment:
if you'd like to use ? here instead of expect we can add returning a Result from main to this PR in another commit! if not, that's OK :)

Reply:
I'm not sure we can do that since [set_hook](https://doc.rust-lang.org/std/panic/fn.set_hook.html) wont accept a `Result` as a return type for the hook. It only accepts `()`.